server/src/main/scala/controllers/Application.scala

```diff
@@ -389,6 +391,33 @@ class Application @Inject()(
       }
   }
 
+  def getDesksAndQueuesCSV(pointInTime: String, terminalName: TerminalName) = Action.async {
+    implicit val timeout: Timeout = Timeout(5 seconds)
+
+    val actor: AskableActorRef = actorSystem.actorOf(
+      Props(classOf[CrunchStateReadActor], SDate(pointInTime.toLong), airportConfig.queues),
+      "crunchStateReadActor" + UUID.randomUUID().toString
+    )
+
+    val potMillidate = MilliDate(pointInTime.toLong)
+    val terminalCrunchResult = actor ? GetCrunchMinutes
```

Comment:
this is called terminalCrunchResult, but it looks like it'll be the whole port's crunch result